docs/topics/proc_show-debug-logging-local-spring-boot.adoc

```diff
@@ -0,0 +1,31 @@
+= Show Debug Logging on Localhost
+
+Use the `JAVA_DEBUG_ENABLED` environment variable when starting your application to enable debug logging.
+
+.Prerequisites
+* An application with debug logging.
+
+.Procedure
+.  Deploy your application to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean fabric8:debug
```

Comment:
So it this on localhost? If you're doing `fabric8:debug`, it's on OpenShift, right?

Reply:
@Ladicek youre right, I was thinking OpenShift/Minishift  for this content. But this should be an example on your local host/pc (non minishift), so will change this example.